### PR TITLE
[DO NOT MERGE] Try the new Github action process

### PR DESCRIPTION
**Pull request by gengliangwang**
Let's verify the new process from https://github.com/apache/spark/pull/32092

### CODE REVIEW

**Comment by gengliangwang** [comment]
GA starts: https://github.com/gengliangwang/spark/actions/runs/747046279

**Comment by dongjoon-hyun** [comment]
Thank you, @gengliangwang . :)

**Comment by dongjoon-hyun** [comment]
BTW, just a question, for the failed GA, only the owner can re-trigger it because it's running on his repo, @HyukjinKwon and @gengliangwang ?

**Comment by HyukjinKwon** [comment]
Ah.. good point. Yeah ..

File: sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala [hunk 1]
```diff
@@ -64,7 +64,7 @@ class SQLQuerySuite extends QueryTest with SharedSparkSession with AdaptiveSpark
       val queryCoalesce = sql("select coalesce(null, 1, '1') from src ")
 
       checkAnswer(queryCaseWhen, Row("1.0") :: Nil)
-      checkAnswer(queryCoalesce, Row("1") :: Nil)
+      checkAnswer(queryCoalesce, Row("2") :: Nil)
```
Comment (HyukjinKwon) [inline sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala:67]:
```suggestion
```

Comment (HyukjinKwon) [inline sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala:67]:
```suggestion
      checkAnswer(queryCoalesce, Row("2") :: Nil)
```

Comment (HyukjinKwon) [reply sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala:67]: In worst case (e.g., author is inactive but PR is ready .. ?), we could push an empty commit to their branch to retrigger .. of course this is an ugly way though

**Comment by gengliangwang** [comment]
> BTW, just a question, for the failed GA, only the owner can re-trigger it because it's running on his repo, @HyukjinKwon and @gengliangwang ?

> In the worst case (e.g., author is inactive but PR is ready .. ?), we could push an empty commit to their branch to retrigger .. of course this is an ugly way though

That's true. 
How about having a backdoor that triggers the test in Apache/Spark upon certain comments? Similar to what we did for Jenkins: "ok to test"

**Comment by HyukjinKwon** [comment]
I'm not sure if that's feasible or not but that would be great to have it.

**Comment by Ngone51** [comment]
I can't see the comment from `github-actions` as it did in https://github.com/HyukjinKwon/spark/pull/34. Is it supposed to comment after testing finished or only in certain cases?

**Comment by HyukjinKwon** [comment]
The comment will be added by "Notify test workflow" job later. There are too many jobs in the queue so it's not triggered yet ...

**Comment by Ngone51** [comment]
Got it, thanks @HyukjinKwon

**Comment by dongjoon-hyun** [comment]
Do we need to test this more, @gengliangwang ?

**Comment by gengliangwang** [comment]
It should be fine now. Let me close it.

**Pull request closed without merging**